lib/prompter.rb

```diff
@@ -1,23 +1,26 @@
+require_relative "font_color_and_style"
+
 class Prompter
-  attr_reader :stdout
+  attr_reader :stdout, :style
 
   def initialize(stdout)
     @stdout = stdout
+    @style = FontColorAndStyle.new
   end
 
   def project_name
-    stdout.puts("\e[0;3mEnter a project name:\e[0m")
+    stdout.puts style.emphasis("Enter a project name:")
```

Comment:
Nice! It is weird on a product level that these prompts are italicized while the others are in orange. Anyway that is the kind of feedback loop you get when you refactor properly. The code starts telling you how the UI is inconsistent.

Reply:
I just went back and looked at the other prompt-ish thing. It was also rendering the menu, so there is some sense but it is odd. Maybe there is a method for the menu prompt styling vs alert??